### ISSUE TITLE
HTTPoison dependency failling on master

### ISSUE THREAD

**Issue by eniolopes** [CargoSense/ex_aws#193]
I'm having the following error pointing to HEAD on master

```
(UndefinedFunctionError) function ExAws.Request.HTTPoison.request/4 is undefined (module ExAws.Request.HTTPoison is not available)
```

I believe the source is https://github.com/CargoSense/ex_aws/blob/master/lib/ex_aws/config/defaults.ex#L10, shouldn't it be using hackney instead?

**Comment by benwilson512** [comment]
It should, that's a bug, PR?

Wonder how tests passed...

**Comment by eniolopes** [comment]
Sure, I can do a PR, what should the default value be? `ExAws.Request.HttpClient` ?

**Comment by eniolopes** [comment]
Ops, I meant `ExAws.Request.Hackne`

**Comment by benwilson512** [comment]
Yup that should do it.

**Comment by benwilson512** [comment]
Fixed by #195
